### ISSUE TITLE
error in  fn bind(&self, sub: &Subscriber<AppModel>

### ISSUE THREAD

**Issue by direstrepo24** [schell/mogwai-realworld#1]
method `bind` has 2 parameters but the declaration in trait `bind` has 3 in:
fn bind(&self, sub: &Subscriber<AppModel>) {
        // bind the nav's output view messages to our input model messages
        sub.subscribe_filter_map(&self.nav.recv, |msg| msg.route().map(|r| AppModel::HashChange {
            route: r.clone()
        }));
    }

**Comment by schell** [comment]
Good catch, thank you! I'm currently quarantined for my family's immigration to New Zealand so I won't be able to get to this for a little while.
